lib/web/websocket/stream/websocketerror.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const { webidl } = require('../../fetch/webidl')
+const { validateCloseCodeAndReason } = require('../util')
+const { kConstruct } = require('../../../core/symbols')
+
+class WebSocketError extends DOMException {
+  #closeCode
+  #reason
+
+  constructor (message = '', init = undefined) {
+    message = webidl.converters.DOMString(message, 'WebSocketError', 'message')
+
+    // 1. Set this 's name to " WebSocketError ".
+    // 2. Set this 's message to message .
+    super(message, 'WebSocketError')
```

Comment:
The message attribute of DomException is not enumerable or so

![image](https://github.com/user-attachments/assets/67e3d493-a9a6-49f3-8a5f-6c979f650516)

I think we want to have that field to be serializable?

```suggestion
    super(message, 'WebSocketError')
    this.message = message
```

Reply:
then we wouldn't be extending DOMException entirely

Reply:
Ok, seems to be consistent with chrome